damus/Views/Buttons/AlbyButton.swift

```diff
@@ -23,7 +23,7 @@ struct AlbyButton: View {
             HStack {
                 Image("alby")
                 
-                Text("Connect to Alby")
```

Comment:
ya'll are too fast. I've already renamed this to `Attach Alby Wallet`

Reply:
No, you're too fast! 😄 It's all good, rebased and added another commit to export strings for translation.